src/inmanta/parser/plyInmantaParser.py

```diff
@@ -1040,7 +1042,25 @@ def myparse(ns: Namespace, tfile: str, content: Optional[str]) -> List[Statement
         return parser.parse(data, lexer=lexer, debug=False)
 
 
+def un_cache(namespace: Namespace, filename: str):
+    cache_filename = filename + "c"
+    if not os.path.exists(cache_filename):
+        return None
+    if os.path.getmtime(filename) > os.path.getmtime(cache_filename):
```

Comment:
```suggestion
    if os.path.getmtime(filename) != os.path.getmtime(cache_filename):
```

Reply:
They will never be exactly the same (the original and the cached version are not created/modified at the same point in time)